### PR TITLE
Us 16

### PR DESCRIPTION
**Pull request by beedawn**
Tests completed.

### CODE REVIEW

File: backend/.DS_Store
Comment (brilzen) [inline backend/.DS_Store:1]: do we need to check in these .DS_Store files

File: backend/auth_tools/.DS_Store
Comment (brilzen) [inline backend/auth_tools/.DS_Store:1]: do we need to check in these .DS_Store files

**Review by brilzen — CHANGES_REQUESTED** [review]
may be add .DS_Store to git ignore files

File: backend/auth_tools/.DS_Store
Comment (beedawn) [reply backend/auth_tools/.DS_Store:1]: Good call, I removed them.

**Review by brilzen — APPROVED** [review]

**Review by tristanhamid — APPROVED** [review]
Sorry for the delay!

**Pull request merged**